pyproject.toml

```diff
@@ -36,6 +38,10 @@ black = {version = "^19.3b0",allows-prereleases = true}
 isort = "^4.3"
 pytest-asyncio = "^0.10.0"
 pytest-cov = "^2.7"
+pytest-django = {version = "^3.5"}
+
+[tool.poetry.extras]
+django = ["django","pytest-django","asgiref"]
```

Comment:
pytest django is only installed when installing dev dependencies, right?

Reply:
As far as I know, but I'm not entirely sure. I'm new to this whole poetry thing 😆